### DISCUSSION TITLE
v6.1.2

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#443] [公告]
## 错误修复

* 修复对 qBittorrentEE shadowBan 的可用性检查总是失败的问题

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v6.1.2'>v6.1.2</a>.</em>

**Comment by Mintisn** [comment]
很遗憾的是在更新或者卸载重装新版本v6.1.2后, 我依然存在之前的问题[#441 PBH无法对支持shadow Ban的qbitEE正常开启shadow ban](https://github.com/PBH-BTN/PeerBanHelper/issues/441)

😭😭😭
